estate/demo/estate_property.xml

```diff
@@ -0,0 +1,81 @@
+<odoo>
+    <record id="umbm_estate_01" model="estate_property">
+        <field name="name">Big Villa 3</field>
+        <field name="state">new</field>
+        <field name="description">A nice and big villa</field>
+        <field name="postcode">12345</field>
+        <field name="estate_property_type" ref="estate.umbm_01"/>
+        <field name="date_availability">2024-04-25</field>
```

Comment:
Please use relative date, in order this demo data to be usable in future versions